easybuild/easyconfigs/e/EPA-ng/EPA-ng-0.3.8-gompi-2022a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'EPA-ng'
+version = '0.3.8'
+
+homepage = 'https://github.com/pierrebarbera/epa-ng'
+description = """
+EPA-ng - Fast, parallel, highly accurate Maximum Likelihood Phylogenetic
```

Comment:
Doesn't this create a new line to start with? E.g. https://github.com/easybuilders/easybuild-easyconfigs/pull/18420/files starts the description right after the `"""`